Felienne/hedy
# Fix incorrect user type metrics #2104

**Please fill out this template by replacing all content between < >**

< Address incorrect user type metrics >

**Description**

<
Fix bug in user type metrics per week.
Fix incorrect week number format causing wrong comparison in Dynamo Between condition
> 

**Fixes < #2104 >**

< Always link the number of the issue or of the discussion that your pr concerns. >
Notable exception: adding content may be done without an accompanying issue 

**How to test**

Run Hedy locally, login as administrator and open the /admin/stats page. Also, ensure there is sample data that you can see.
- Check that all options for selecting period work: This week, past 2 weeks, past 4 weeks and past year. They should not result in 500 errors and they should return correct values (which highly depends on the sample data you generated).
- Go to the charts with names **User type per level** and **User type per week**. Ensure there are 4 series: anonymous, logged-in, student and unknown user types. The unknown user type refers to the previous data collected before we differentiated user types.
- The graphs **User type per level** and **User type per week** should show equal total number of users.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
